src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
-
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blur',
 }
 
-export class Triangle {
+enum Shape {
+  Triangle = 'Triangle',
+  Circle = 'gCircle',
+  Rectangle = 'Rectangle',
+}
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  constructor(
+    public shape: Shape,
```

Comment:
Shape should not be in consturctor, cuz you may create `new  Triangle('gCircle' ...)` fix all classes
```suggestion
export class Triangle implements Figure {
  shape = Shape.Triangle;

  constructor(
```